### ISSUE TITLE
New version in hex

### ISSUE THREAD

**Issue by mrcnkoba** [chvanikoff/apns4ex#58]
Hey guys!

I see only version 0.9.4 in hex, could you please publish the new version? 😄

**Comment by ghost** [comment]
Would be great to be able to use poison 3

**Comment by zepplock** [comment]
Please make a PR with new Poison, I'll update version and ask @chvanikoff to push a new version out. Unfortunately I don't have rights for `hex.pm`

**Comment by mrcnkoba** [comment]
Oh 22 June, long time no see!

**Comment by zepplock** [comment]
@mrcnkoba I don't have any Elixir code in production right now (unfortunately). But I don't want to drop support for the library if it is being used. In case @chvanikoff does not reply, I'll republish it to hex under a different name/version (maybe `apns5ex` ;)

**Comment by mrcnkoba** [comment]
@zepplock That would be a great idea. I'm genuinely loving the new name 👍

**Comment by nkezhaya** [comment]
@zepplock Nothing yet? :(

**Comment by zepplock** [comment]
Unfortunately I have no Elixir env anymore and no iOS project to verify the changes. So @chvanikoff is your best bet.

**Comment by nkezhaya** [comment]
@zepplock It might be worthwhile to publish a new Hex version. It seems like this repo is no longer maintained.

**Comment by zepplock** [comment]
I agree, I would gladly do so but I have no rights to publish, only @chvanikoff can since hex does not allow non-owners to publish.

**Comment by nkezhaya** [comment]
Sorry, I meant a new Hex package.

(Alternatively, switch to [Pigeon](https://github.com/codedge-llc/pigeon)...)
